scripts/generate-changelog.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+
+const replace = require('replace-in-file')
+const util = require('util')
+const exec = util.promisify(require('child_process').exec)
+const spawn = util.promisify(require('child_process').spawn)
+
+const name = require('emoji-name-map')
+
+const lernaConfig = require('../lerna.json')
+
+const CHANGELOG_FILE = 'CHANGELOG.md'
+
+async function main() {
+  const lastHashCmd = await exec('git rev-list --tags --max-count=1')
+  if (lastHashCmd.stderr) {
+    throw lastHashCmd.stderr
+  }
+
+  const oldTagCmd = await exec(`git describe --tags ${lastHashCmd.stdout}`)
+  if (oldTagCmd.stderr) {
+    throw oldTagCmd.stderr
+  }
+  const oldTag = oldTagCmd.stdout.replace(/\n$/, '')
+
+  const commitsCmd = await exec(`git log ${oldTag}..HEAD --pretty=format:"- %s"`)
+  if (commitsCmd.stderr) {
+    throw commitsCmd.stderr
+  }
+
+  const changesWithEmojis = await emojiNameToUnicode(
+    `# Changelog\n\n## v${lernaConfig.version}\n\n${commitsCmd.stdout}`
+  )
+
+  const changesWhihPullRequestLinks = changesWithEmojis.replace(
+		/\(#(\d+)\)/gm,
+    (match, id) => `[(#${id})](https://github.com/DataDog/browser-sdk/pull/${id})`
```

Comment:
```suggestion
    (match, id) => `([#${id}](https://github.com/DataDog/browser-sdk/pull/${id}))`
```
it would be more consistent with usual github PR number link decoration

Reply:
Done